tinygrad/tensor.py

```diff
@@ -309,10 +309,9 @@ def min(self, axis=None, keepdim=False): return -((-self).max(axis=axis, keepdim
   def mean(self, axis=None, keepdim=False):
     out = self.sum(axis=axis, keepdim=keepdim)
     return out * (prod(out.shape)/prod(self.shape))
-  # TODO: implement unbiased True option for torch bessel's correction (subtracting 1 from divisor causes 0.01 error)
-  def std(self, axis=None, keepdim=False):
+  def std(self, axis=None, keepdim=False, unbiased=False):
```

Comment:
Please add tests

Reply:
You mean somtething like that for std: 
helper_test_op([(45, 65, 85)], lambda x: torch.std(x, unbiased=True), lambda x: Tensor.std(x, unbiased=True))  ?

Reply:
The test should already be included in test_std here: 7b689987182fedd482cc8879fe3935a12d813530